src/registry/MergedRegistry.ts

```diff
@@ -51,6 +59,11 @@ export class MergedRegistry extends BaseRegistry implements IRegistry {
     return (await this.getAddresses())[chainName] || null;
   }
 
+  async getChainLogoUri(chainName: ChainName): Promise<string | null> {
+    const results = await this.multiRegistryRead((r) => r.getChainLogoUri(chainName));
+    return results.filter((uri) => uri !== null)[0] || null;
```

Comment:
nit: definitely don't feels strongly but could use `.find` which short circuits upon a truthy value

Reply:
Ah yes should have thought of that thanks